Add Postgres dockerfile

ttung opened the pull request:
### Description
This is pulled from https://github.com/docker-library/postgres/commit/e8cb2acbb372f0c89219792f82b0bdd2e405602f

### Test plan
Used in subsequent PR

phoenixAja commented on docker/postgres/Dockerfile:
```diff
@@ -0,0 +1,182 @@
+		dpkg-dev dpkg \
+		flex \
+		gcc \
+#		krb5-dev \
```
should keep this?

phoenixAja commented on docker/postgres/Dockerfile:
```diff
@@ -0,0 +1,182 @@
+		linux-headers \
+		llvm10-dev clang g++ \
+		make \
+#		openldap-dev \
```
same

phoenixAja commented on docker/postgres/Dockerfile:
```diff
@@ -0,0 +1,182 @@
+		perl-utils \
+# configure: error: Perl module IPC::Run is required to run TAP tests
+		perl-ipc-run \
+#		perl-dev \
+#		python-dev \
+#		python3-dev \
+#		tcl-dev \
```
same

phoenixAja commented on docker/postgres/Dockerfile:
```diff
@@ -0,0 +1,182 @@
+		$runDeps \
+		bash \
+		su-exec \
+# tzdata is optional, but only adds around 1Mb to image size and is recommended by Django documentation:
+# https://docs.djangoproject.com/en/1.10/ref/databases/#optimizing-postgresql-s-configuration
```
is it also recommended by flask?

ttung commented:
I'm just importing their dockerfile verbatim.  I only plan on making the minimal set of changes to get what I want in #135.

phoenixAja commented on docker/postgres/docker-entrypoint.sh:
```diff
@@ -0,0 +1,2482 @@
+
```
is this the right file? It looks like it's all html

phoenixAja commented on docker/postgres/Dockerfile:
```diff
@@ -0,0 +1,182 @@
+#
+STOPSIGNAL SIGINT
+#
+# An additional setting that is recommended for all users regardless of this
+# value is the runtime "--stop-timeout" (or your orchestrator/runtime's
+# equivalent) for controlling how long to wait between sending the defined
+# STOPSIGNAL and sending SIGKILL (which is likely to cause data corruption).
+#
+# The default in most runtimes (such as Docker) is 10 seconds, and the
+# documentation at https://www.postgresql.org/docs/12/server-start.html notes
+# that even 90 seconds may not be long enough in many instances.
```
should we add this setting?

phoenixAja commented:
> I'm just importing their dockerfile verbatim. I only plan on making the minimal set of changes to get what I want in #135.

ok i'll just approve then lol

Commit:
Add Postgres dockerfile

### Description
This is pulled from https://github.com/docker-library/postgres/commit/e8cb2acbb372f0c89219792f82b0bdd2e405602f

### Test plan
Used in subsequent PR
